docs/platform/howto/create_new_service_user.rst

```diff
@@ -3,6 +3,15 @@
 
 Service users are users that only exist in the scope of the corresponding Aiven service. They are unique to this service and not shared with any other services, and can be granted restricted permissions compared to the default ``avnadmin`` user. You can add service users for all the Aiven services, with the exception of Aiven for Apache Flink® and Aiven for Grafana®.
 
+.. warning::
+   By default, the maximum amount of users allowed on a service is 50. If a service user is created that exceeds the max users, the following error will be produced.
```

Comment:
```suggestion
   By default, the maximum amount of users allowed on a service is 50. 
```

Reply:
For this and the code block, I would like to keep these if possible. The main reason for this is that customers will file support cases with us saying they got the error. If we include an example and description of the error here, customers would be able to find it through google, and would be better informed on what they need to ask for in the case. Let me know your thoughts on that.